Resources/Prototypes/Roles/Jobs/Cargo/quartermaster.yml

```
@@ -15,7 +15,7 @@
   weight: 10
   startingGear: QuartermasterGear
   icon: "QuarterMaster"
-  supervisors: job-supervisors-hop
+  supervisors: job-supervisors-captain #corvax-hierarchy
```

Потом же не найти ничего
```suggestion
  supervisors: job-supervisors-captain # Corvax-Hierarchy
```

И это лучше сделать в апстрим

---

У них разве тоже км глава отдельный?